### ISSUE TITLE
Can't install Genghis on Mac. [Mavericks]

### ISSUE THREAD

**Issue by LeCoupa** [bobthecow/genghis#170]
I have Xcode and the command line tools installed. But when running "sudo gem install genghisapp", the following error appears:

```
ERROR:  Error installing genghisapp:
    ERROR: Failed to build gem native extension.

    /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby extconf.rb
creating Makefile

make "DESTDIR="
compiling generator.c
linking shared-object json/ext/generator.bundle
clang: error: unknown argument: '-multiply_definedsuppress' [-Wunused-command-line-argument-hard-error-in-future]
clang: note: this will be a hard error (cannot be downgraded to a warning) in the future
make: *** [generator.bundle] Error 1


Gem files will remain installed in /Library/Ruby/Gems/2.0.0/gems/json-1.8.1 for inspection.
Results logged to /Library/Ruby/Gems/2.0.0/gems/json-1.8.1/ext/json/ext/generator/gem_make.out
```

What should I do? I'm on Mavericks.

**Comment by bobthecow** [comment]
That's a good question. I don't use the system Ruby on my Mavericks box, so I've never run into this.

It appears that the problem is with compiling the `json` gem. Since Mavericks has Ruby 2.x, it already _has_ JSON built in, so I'm not sure why it would even need to compile the gem :-/

I'll look into this.

**Comment by rwilcox** [comment]
The bug in the json project: flori/json#200

**Comment by bobthecow** [comment]
@rwilcox :+1: Thanks for the link.

**Comment by LeCoupa** [comment]
Thank you!

**Comment by sodabrew** [comment]
Hey all, this was resolved by OS X 10.9.3 and by XCode 5.1.1 without any system file patching.

**Comment by bobthecow** [comment]
Awesome :)
